api/src/main/java/com/capitalone/dashboard/auth/AuthProperties.java

```diff
@@ -0,0 +1,70 @@
+package com.capitalone.dashboard.auth;
+
+import java.util.UUID;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "auth")
+public class AuthProperties {
+	
+	private static final Logger LOGGER = LoggerFactory.getLogger(AuthProperties.class);
+	
+	private Long expirationTime;
+	private String secret;
```

Comment:
Correct, you have the ability to set the signing key for the JWT token, this gives you the ability to set the same signing key on multiple instances of the API in case you had multiple running for load balancing.  This would make sure that a request could be sent to any instance of the API and be validated properly with the key.